classes/cluster/mk20_stacklight_advanced/openstack/compute.yml

```diff
@@ -3,6 +3,7 @@ classes:
 - system.linux.system.repo.mos8
 - system.nova.compute.cluster
 - system.opencontrail.compute.cluster
+- service.opencontrail.compute.cluster
```

Comment:
@cznewt service.opencontrail.compute.cluster is only available in the stacklight branch for now. wouldn't it be a problem if it's included for all labs?